configuration/src/main/java/io/camunda/configuration/beanoverrides/BrokerBasedPropertiesOverride.java

```diff
@@ -671,8 +671,13 @@ private void populateCamundaExporter(final BrokerBasedProperties override) {
     setArg(args, "connect.url", database.getUrl());
     setArg(args, "connect.clusterName", database.getClusterName());
     setArg(args, "connect.dateFormat", database.getDateFormat());
-    setArg(args, "connect.socketTimeout", database.getSocketTimeout());
-    setArg(args, "connect.connectTimeout", database.getConnectionTimeout());
+    final var socketTimeout = database.getSocketTimeout();
+    setArg(args, "connect.socketTimeout", socketTimeout != null ? socketTimeout.toMillis() : null);
```

Comment:
is that config nullable?
if we pass `null` as input, what does the getter return?

I'll approve this in the meantime, but let's check this